src/queries.js

```diff
@@ -56,8 +57,7 @@ const repoSubQuery = (before = new Date(), after, commits, reactionsInQuery) =>
   const masterCommits = node('ref', { qualifiedName: 'refs/heads/master' }, [
     typeSwitch('target', {}, [
       ['Commit', [
-        edge('history', { since: a, until: b },
-          [commitAuthorQ, val('committedDate')])
+        edge('history', { since: a, until: b }, [commitAuthorQ, val('committedDate')])
```

Comment:
`standard` still show some weird errors which I didn't manage to fix for line 61.

Reply:
Did you try standard --fix?

Reply:
Yes, but (to my surprise) it didn't do __any__thing to fix those two errors (probably a bug?).

Reply:
Fixed.